src/applications/mhv-medications/tests/containers/PrescriptionsPrintOnly.unit.spec.jsx

```diff
@@ -9,6 +9,21 @@ import PrescriptionsPrintOnly from '../../containers/PrescriptionsPrintOnly';
 let sandbox;
 
 describe('Medications List Print Page', () => {
+  const setupWithError = (params = {}) => {
+    const hasError = {
+      status: 404,
+      message: 'No static resource my_health/v1/medical_records/allergies.',
+    };
+    return renderWithStoreAndRouterV6(
+      <PrescriptionsPrintOnly hasError={hasError} />,
+      {
+        initialState: {},
+        reducers,
+        initialEntries: ['/?page=1'],
+        ...params,
+      },
+    );
+  };
```

Comment:
Since this setup function is only used once, could you make it part of the `it block?